code/lib/preview-api/src/modules/store/csf/prepareStory.ts

```diff
@@ -149,6 +154,23 @@ export function prepareStory<TRenderer extends Renderer>(
   };
 }
 
+export const getTriangularNumber = (seed: number) => (seed * (seed + 1)) / 2;
+
+export const delayForCanvas = async <TRenderer extends Renderer>(
+  canvasElement: TRenderer['canvasElement']
+) => {
+  let backOff = 0;
+  // @ts-expect-error Renderer uses canvasElement: unknown, but we know it's a div.
```

Comment:
Obviously using `@ts-expect-error` is less than ideal. What _would_ be desirable as `Renderer` is largely full of unknowns? What support targets should I be considering and testing?